soh/src/overlays/actors/ovl_En_Jsjutan/z_en_jsjutan.c

```diff
@@ -402,7 +402,7 @@ void EnJsjutan_Draw(Actor* thisx, GlobalContext* globalCtx2) {
         this->unk_164 = false;
         u8* carpTex = ResourceMgr_LoadTexByName(sCarpetTex);
         u8* shadTex = sShadowTex;
-        for (i = 0; i < ARRAY_COUNT(shadTex); i++) {
```

Comment:
I have no idea why this worked before (IDO being bad?). But for the array size to be calculated correctly, it needed to be changed to something that was, well, an array